pkg/components/metallb/manifests.go

```diff
@@ -371,13 +365,19 @@ spec:
             - ALL
           readOnlyRootFilesystem: true
       hostNetwork: true
+      # XXX: Lokomotive spcific change.
```

Comment:
I don't know if it's worth worrying about, but I've used a different phrasing here: https://github.com/kinvolk/lokomotive/blob/fa7f06a6a9357244458856b1bd1e2518f79661e4/pkg/components/metallb/manifests.go#L289

I'll leave this up to you.